modules/service-catalog/app-registry/deployspec.yaml

```diff
@@ -0,0 +1,25 @@
+publishGenericEnvVariables: true
+deploy:
+  phases:
+    install:
+      commands:
+      - npm install -g aws-cdk@2.70.0
+      - pip install -r requirements.txt
+    build:
+      commands:
+      - cdk deploy --require-approval never --progress events --app "python app.py" --outputs-file ./cdk-exports.json
+      # Export metadata
+      - export SEEDFARMER_MODULE_METADATA=$(python -c "import json; file=open('cdk-exports.json'); print(json.load(file)['${SEEDFARMER_PROJECT_NAME}-${SEEDFARMER_DEPLOYMENT_NAME}-${SEEDFARMER_MODULE_NAME}']['metadata'])")
```

Comment:
why not start using the helper commands?

Reply:
`- seedfarmer metadata convert -f cdk-exports.json || true`

Reply:
because i load the env var `SEEDFARMER_MODULE_METADATA` in my python script, and grab the AppRegistry details